creator/lib/screens/home_screen.dart

```diff
@@ -110,35 +126,55 @@ class _HomeScreenState extends State<HomeScreen> {
     return Column(
       mainAxisAlignment: MainAxisAlignment.spaceEvenly,
       children: [
-        SizedBox(height: 24),
+        SizedBox(
+          height: 24,
+        ),
         Center(
-          child: Text(
-            titleList[index],
-            style: TextStyle(
-                fontWeight: FontWeight.bold, color: Colors.white, fontSize: 16),
-          ),
+          child: _mainMessageOfSalonTopic(index),
+        ),
+        SizedBox(
+          height: 50,
         ),
-        SizedBox(height: 50),
         Center(
           child: Padding(
-            padding: const EdgeInsets.all(12.0),
-            child: Text(
-              subList[index],
-              style: TextStyle(
-                  fontWeight: FontWeight.bold,
-                  color: Colors.white,
-                  fontSize: 8),
+            padding: const EdgeInsets.all(
+              12.0,
             ),
+            child: _subMessageOfSalonTopic(index),
           ),
         ),
       ],
     );
   }
 
+  Widget _mainMessageOfSalonTopic(int index) {
+    return Text(
+      titleList[index],
+      style: TextStyle(
```

Comment:
please use Theme instead of defining the style every time. You can check Gospel Crowd for a reference implementation

Reply:
I fixed it.